src/docs/modules/ROOT/pages/modeling-guidelines.adoc

```diff
@@ -324,16 +324,17 @@ value is the recorded value. In order to describe these semantics the shared `Ti
 Characteristic is used. This is a subclass of the `SortedSet` Characteristic and defines that the
 `dataType` must be a subclass of the `TimeSeriesEntity`.
 
-The `xref:entities.adoc#time-series-entity[TimeSeriesEntity]` is declared as an Abstract Entity. It defines the
-`timestamp` Property with the `Timestamp` Characteristic, which describes values containing timestamps conforming
-to `xref:datatypes.adoc#data-types[xsd:dateTime]`, a date and time with or without time zone. No `value` Property is
-defined, hence the semantics of the value can be defined in the specific Aspect Model.
+The `xref:entities.adoc#time-series-entity[TimeSeriesEntity]` is declared as an Abstract Entity. It
+defines the `timestamp` Property with the `Timestamp` Characteristic, which describes values
+containing timestamps conforming to `xref:datatypes.adoc#data-types[xsd:dateTime]`, a date and time
+with or without time zone. No `value` Property is defined, hence the semantics of the value can be
```

Comment:
IMHO this is misleading. A `value` Property is defined as an Abstract Property. We should explicitly mention this.